backend/oma/src/main/java/com/oma/dao/CompanyDAOImplementation.java

```diff
@@ -16,10 +16,9 @@
     SessionFactory sessionFactory;
 
     @Override
-    @Transactional
     public void save(Company company) {
         Session factoryCurrentSession = sessionFactory.getCurrentSession();
-        factoryCurrentSession.save(company);
+        factoryCurrentSession.saveOrUpdate(company);
```

Comment:
Not sure if it good idea, save is not the same as update, or what you think if we use sql base?